src/pages/AdminPage.jsx

```diff
@@ -0,0 +1,213 @@
+            />
+            {formErrors.name && <span className="error">{formErrors.name}</span>}
+          </div>
+          <div className='from-fields'>
+            <label>Product Price:</label>
+            <input
+              type="text"
+              value={productPrice}
+              onChange={(e) => setProductPrice(e.target.value)}
+            />
+            {formErrors.price && <span className="error">{formErrors.price}</span>}
+          </div>
+          <div className='from-fields'>
+            <label>Product Picture URL:</label>
+            <input
+              type="text"
+              value={productPicture}
+              onChange={(e) => setProductPicture(e.target.value)}
+            />
+            {formErrors.picture && <span className="error">{formErrors.picture}</span>}
+          </div>
+          <div className='from-fields'>
+            <label>Product CPU:</label>
+            <input
+              type="text"
+              value={productCPU}
+              onChange={(e) => setProductCPU(e.target.value)}
+            />
+            {formErrors.cpu && <span className="error">{formErrors.cpu}</span>}
+          </div>
+          <div className='from-fields'>
+            <label>Product Memory:</label>
+            <input
+              type="text"
+              value={productMemory}
+              onChange={(e) => setProductMemory(e.target.value)}
+            />
+            {formErrors.memory && <span className="error">{formErrors.memory}</span>}
+          </div>
+          <div className='from-fields'>
+            <label>Product Storage:</label>
+            <input
+              type="text"
+              value={productStorage}
+              onChange={(e) => setProductStorage(e.target.value)}
+            />
+            {formErrors.storage && <span className="error">{formErrors.storage}</span>}
+          </div>
+          <button onClick={handleAddProduct}>Add Product</button>
+        </div>
+        <div className='admin-page-show'>
+          <div className='admin-page-search'>
+            <h2>Search Product</h2>
+            <input
+              type="text"
+              placeholder="Enter Keyword"
+              value={searchKeyword}
+              onChange={(e) => setSearchKeyword(e.target.value)}
+            />
+          </div>
+          <div className='admin-page-list'>
+            <h2>Product List</h2>
+            <ul>
+              {products.filter((product) => {
+                if (searchKeyword === '') {
+                  return product;
+                }
+                if (product.name.toLowerCase().includes(searchKeyword.toLowerCase())) {
+                  return product;
+                }
+                return null;
+              }).map((product) => (
+                <li key={product.id}>
+                  <p>{product.name}</p>
+                  <button onClick={() => handleDeleteProduct(product.id)}>Delete</button>
+                </li>
```

Comment:
These delete buttons are not aligned when product names are of different length.

Reply:
@shasherazi What! Can I have screenshot what are seeing?

Reply:
![image](https://github.com/shasherazi/laptops-ecommerce/assets/63230301/1b1d690c-14c1-4445-9222-8efc45070715)